vue-components/.eslintrc.js

```diff
@@ -1,29 +1,117 @@
 module.exports = {
-  root: true,
-  env: {
-    node: true
-  },
-  extends: [
-    'plugin:vue/essential',
-    '@vue/standard',
-    '@vue/typescript/recommended'
-  ],
-  parserOptions: {
-    ecmaVersion: 2020
-  },
-  rules: {
-    'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
-    'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off'
-  },
-  overrides: [
-    {
-      files: [
-        '**/__tests__/*.{j,t}s?(x)',
-        '**/tests/unit/**/*.spec.{j,t}s?(x)'
-      ],
-      env: {
-        jest: true
-      }
-    }
-  ]
-}
+	extends: [
+		'wikimedia',
+		'wikimedia/node',
```

Comment:
I think it still makes sense to use `wikimedia/node` for `babel.config.js`, `jest.config.js` and the likes because those are actually run in a node context, and are different from the production code in that they use common.js `module.exports` etc